schemas/v1/6_restore_prove_aggregate_param_schema.go

```diff
@@ -0,0 +1,36 @@
+package v1
+
+func init() {
+	patches.Register(
+		6,
+		`
+	-- Fix ProveCommitAggregate params schema change from Lily v0.8.6
+	-- by updating incorrect JSONB in-place for the affected epochs
+	-- (context: https://github.com/filecoin-project/lily/issues/904)
+
+	SELECT jsonb_set(
+
+		-- given value in column "params"
+		params,
+
+		-- modify the value at key "SectorNumbers"
+		'{SectorNumbers}'::text[],
+
+		-- to use this interpolated json literal
+		concat(
+			'{"rle":', params->'SectorNumbers'->>'RLE',
+			',"elemcount":', params->'SectorNumbers'->>'Count',
+			',"_type":"bitfield"}'
+		)::jsonb,
```

Comment:
This creates a JSON literal which is cast to a `jsonb`. It interpolates the existing values into the schema with the original keys.